proto/zetachain/zetacore/pkg/chains/chains.proto

```diff
@@ -48,6 +48,8 @@ enum ChainName {
   solana_mainnet = 21;
   solana_devnet = 22;
   solana_localnet = 23;
+
+  btc_signet_testnet = 24;
```

Comment:
same

Reply:
I considered that , but wanted to keep `testnet` in the name to be consistent with other networks, which follow the name patters network_netoworkType